<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dictionary</class>
 <widget class="QWidget" name="Dictionary">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>517</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dictionary</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Dictionaries</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnAdd">
       <property name="minimumSize">
        <size>
         <width>37</width>
         <height>18</height>
        </size>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/images/images/accept.png</normaloff>:/images/images/accept.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="btnDelete">
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/images/images/close.png</normaloff>:/images/images/close.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="4">
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QListWidget" name="lstDictionary">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
       <widget class="LAP_DataGrid" name="tblDictionary">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>20</number>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LAP_DataGrid</class>
   <extends>QTableWidget</extends>
   <header location="global">lap_datagrid.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lstDictionary</tabstop>
  <tabstop>tblDictionary</tabstop>
  <tabstop>btnAdd</tabstop>
  <tabstop>btnDelete</tabstop>
 </tabstops>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
